web-common/src/features/dashboards/stores/filter-utils.ts

```diff
@@ -359,6 +377,18 @@ export function isJoinerExpression(expression: V1Expression | undefined) {
   );
 }
 
+export function unwrapRedundantJoinerExpression(
+  expression: V1Expression | undefined,
+) {
+  if (
+    !expression ||
+    !isJoinerExpression(expression) ||
+    (expression.cond?.exprs?.length && expression.cond.exprs.length > 1)
+  )
+    return expression;
+  return expression.cond?.exprs?.[0];
+}
```

Comment:
This is fairly clear, but I'm wondering if this implies that we have a problem elsewhere in the code? And what's a "joiner" expression?

Reply:
The type supports multiple measure filter per dimension. That is why there is always an and at the top level. Moves things around and added comments to make things clearer.

Reply:
If there's only 1 measure (or any other) filter, then we shouldn't have a wrapper AND expression. (That may be out-of-scope for this PR– if that's the case, so be it.)